ae/templates/engine-service.yaml

```diff
@@ -24,6 +24,12 @@ spec:
     - port: 5701
       name: {{ .Values.engine.name }}-hz
       protocol: TCP
+    {{- if .Values.engine.services.core.service.enabled }}
+    - port: {{ .Values.engine.services.core.service.externalPort }}
+      targetPort: {{ .Values.engine.services.core.http.port }}
+      protocol: TCP
+      name: {{ .Values.engine.name }}-techapi
+    {{- end }}
```

Comment:
I don't get this part 🧐
What is does mean `.Values.engine.services.core.service.enabled` ? Which "service" is enabled here ? Should it be `.Values.engine.services.core.service.http.enabled` instead ?

Same for `.Values.engine.services.core.service.externalPort` ?

Reply:
Ok it refers to the "core" service which corresponds to the technical api itself. The "http" is only a way to expose the technical api (it is the only supported for now). So makes sense to keep it like this and keep consistency with other products (apim, am). 👍